.NET/Microsoft.Recognizers.Text.DataDrivenTests/Sequence/TestSequence_English.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Recognizers.Text.DataDrivenTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Recognizers.Text.Sequence.Tests
+{
+    [TestClass]
+    public class TestSequence_English : TestBase
+    {
+        public static TestResources TestResources { get; protected set; }
+        public static IDictionary<string, IModel> Models { get; protected set; }
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            TestResources = new TestResources();
+            TestResources.InitFromTestContext(context);
+            Models = new Dictionary<string, IModel>();
+        }
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            base.TestSpecInitialize(TestResources);
+            base.ModelInitialize(Models);
+        }
+
+        [DataSource("Microsoft.VisualStudio.TestTools.DataSource.CSV", "PhoneNumberModel-English.csv", "PhoneNumberModel-English#csv", DataAccessMethod.Sequential)]
+        [TestMethod]
+        public void PhoneNumberModel()
+        {
+            TestNumber();
```

Comment:
why TestNumber?

Reply:
I think the Sequence's check process is like the Number one, so I just reuse it here. Should I create a new function with the same code or create a function to wrap it.